node/pkg/logscribeconsumer/types.go

```diff
@@ -0,0 +1,72 @@
+package logscribeconsumer
+
+import (
+	"time"
+
+	"bisonai.com/orakl/node/pkg/logscribe/api"
+	"github.com/rs/zerolog"
+)
+
+const (
+	DefaultLogStoreInterval = 10 * time.Second
+	DefaultBufferSize       = 3000
+	MinimalLogStoreLevel    = zerolog.WarnLevel
```

Comment:
MinimalLogStoreLevel should probably be now something like
`DefaultLogStoreLevel`, and the logLevel passed when init should be prioritized